.github/workflows/lint.yml

```
@@ -22,7 +22,15 @@ jobs:
       - name: Ensure consistent CircleCI YAML config
         run: cd .circleci && ./ensure-consistency.py
       - name: Ensure consistent GHA workflows in cancel_redundant_workflows.yml
-        run: .github/scripts/ensure_consistent_workflows.py
+        run: |
+          pip install ruamel.yaml==0.17.4
+          echo "Please locally run .github/scripts/regenerate_cancel_redundant_workflow.py and commit if this step fails."
+          .github/scripts/regenerate_cancel_redundant_workflow.py
+          set -eux
+          CHANGES=$(git status --porcelain)
+          echo "$CHANGES"
+          git diff
+          [ -z "$CHANGES" ]
```

since you really just want to check the one file here, I'd recommend simplifying this:
```suggestion
          git diff --exit-code .github/workflows/cancel_redundant_workflows.yml
```

---

Ah that's cool!